modules/manage/pages/kubernetes/storage/resize-persistentvolumes.adoc

```diff
@@ -0,0 +1,141 @@
+= Expand PersistentVolumes
+:url-kubernetes-expand-pvc: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims
+:description: Learn how to expand the size of your PersistentVolumes to give Redpanda brokers more storage capacity.
+
+To give Redpanda brokers more storage, you can expand the size of PersistentVolumes. The way you expand PersistentVolumes depends on the provisioner that you use.
+
+== Expand PersistentVolumes that use static provisioners
+
+The process for resizing PersistentVolumes that use a static provisioner varies depending on the way your file system is allocated. Follow the recommended process for your system. You do not need to make any configuration changes to the Redpanda Helm chart or your Redpanda custom resource.
+
+== Expand PersistentVolumes that use dynamic provisioners
+
+To expand a PersistentVolume that uses a dynamic provisioner, make sure that your StorageClass is capable of volume expansions. For a list of volumes that support volume expansion, see the {url-kubernetes-expand-pvc}[Kubernetes documentation^].
+
+=== Expand PersistentVolumes for the Redpanda data directory
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    storage:
+      persistentVolume:
+        enabled: true
+        size: <custom-size>Gi
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`persistentvolume-size.yaml`
+[,yaml]
+----
+storage:
+  persistentVolume:
+    enabled: true
+    size: <custom-size>Gi
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --values persistentvolume-size.yaml --reuse-values
```

Comment:
I wonder if we have Statefulset with already existing Pods, I don't think we could change size of the PVC. You could change the specification, but only new Pods would have different PVC size.

Existing PVC would need to be changed outside of helm chart.

Reply:
@RafalKorepta what should be the change here?

Reply:
Good question. I think we could mention about the need for user manual intervention in PersistentVolume and PersistentVolumeClaim to change the size value. Not all storage drivers would pick such change and act upon. I would need to verify which can support that operation.

Reply:
cc @joejulian because I believe you helped with the original content in https://docs.redpanda.com/current/manage/kubernetes/configure-storage/#resize-persistentvolumes

Reply:
@RafalKorepta I added this info to the doc.

Reply:
Thank you